.github/workflows/deploy-move-package.yml

```diff
@@ -0,0 +1,86 @@
+name: Deploy Move package
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Name of branch to checkout'
+        required: true
+      network:
+        description: 'Network to deploy to'
+        required: true
+      faucet:
+        description: 'TBD'
+        required: true
+      script-path:
+        description: 'Script to tun'
+        required: true
+      dry-run:
+        description: 'Run in dry-run mode'
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+  deploy-move-package:
+    environment: release
+    name: TBD
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out the repo
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.inputs.branch }}
+
+    - name: Start iota sandbox
+      uses: './.github/actions/iota-rebase-sandbox/setup'
+      with:
+        platform: 'linux'
+        iota-version: ${{ github.event.inputs.network }}
+        start-sandbox: false
+
+    - name: Setup Keytool
+      uses: './.github/actions/iota-rebase-sandbox/load-keytool'
+      with:
+        network: ${{ github.event.inputs.network }}
+        pk: ${{secrets.TRUST_FRAMEWORK_PRODUCTS_PRIVATE_KEY}}
+        bak_pk: ${{secrets.TRUST_FRAMEWORK_PRODUCTS_PRIVATE_KEY_BACKUP}} #only for testing, TODO: remove
```

Comment:
Should this be removed before merging or in a follow-up PR? ^^

Reply:
We can only test it once merged, so let's remove in a follow up PR 👌